sdk/monitor/opentelemetry-exporter/test/common/scenario/basic.ts

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as opentelemetry from "@opentelemetry/api";
+import { BasicTracerProvider, SimpleSpanProcessor } from "@opentelemetry/tracing";
+import { AzureMonitorTraceExporter } from "../../../src";
+import { Expectation, Scenario } from "./types";
+import { promisify } from "util";
+import { Envelope, RemoteDependencyData, RequestData } from "../../../src/Declarations/Contracts";
+import { msToTimeSpan } from "../../../src/utils/breezeUtils";
+import { CanonicalCode } from "@opentelemetry/api";
+
+const sleep = promisify(setTimeout);
```

Comment:
I've never thought of this! If you want it to also work in the browser, we have a `delay` function in `@azure/core-http` that does this without promisify

Reply:
Replaced with `delay`. It was drop-in compatible!